conf/log4j2.yaml

```diff
@@ -0,0 +1,80 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+
+Configuration:
+  status: INFO
+  monitorInterval: 30
+  name: pulsar
+
+  # Example: logger-filter script
+  Scripts:
+    ScriptFile:
+      name: filter.js
+      language: JavaScript
+      path: ./conf/log4j2-scripts/filter.js
+      charset: UTF-8
+  
+  Appenders:
+  
+    # Console 
+    Console:
+      name: Console
+      target: SYSTEM_OUT
+      PatternLayout:
+        Pattern: "%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"
+        
+   # Rolling file appender configuration
+    RollingFile:
+      name: RollingFile
+      fileName: "${sys:pulsar.log.dir}/${sys:pulsar.log.file}"
+      filePattern: "/${sys:pulsar.log.file}-%d{MM-dd-yyyy}-%i.log"
```

Comment:
> is it writing on `logs/..`

Yes, If i start `pulsar standalone` and `pulsar-daemon start broker` then default it creates and writes to `logs/pulsar-standalone.log` and `logs/pulsar-broker-myhost.log`.

>  Here we used to have ${pulsar.log.dir}/${pulsar.log.file}

Actually, in log4j2 [system-properties-lookup](https://logging.apache.org/log4j/2.0/manual/lookups.html#SystemPropertiesLookup) happens with prefix `sys:`. It doesn't work if I remove that prefix.